deps/libgit2.mk

```diff
@@ -70,10 +70,14 @@ CURL_LOCATION=$(shell which curl 2>/dev/null)
 WGET_LOCATION=$(shell which wget 2>/dev/null)
 ifeq ($(shell [ -x $(CURL_LOCATION) ] && $(CURL_LOCATION) -V >/dev/null && echo exists),exists)
 $(shell curl --cacert $(OPENSSL_DIR)/certs/GlobalSign_Root_CA.pem -O https://curl.haxx.se/ca/cacert.pem && \
+curl --cacert $(OPENSSL_DIR)/certs/GlobalSign_Root_CA.pem -O https://curl.haxx.se/ca/cacert.pem.sha256 && \
```

Comment:
better than nothing, but downloading the checksum from the same site doesn't help with repeatability of the build - is there a versioned source of these?